<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoEntreprise</class>
 <widget class="QDialog" name="InfoEntreprise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="TitreLabel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>The Gestion</string>
   </property>
  </widget>
  <widget class="QFrame" name="InfoWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>241</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">&quot;border: 1px solid black; border-radius: 10px;&quot;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Mon Compte</string>
    </property>
   </widget>
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="SurnameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Prenom</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatutLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Statut</string>
    </property>
   </widget>
   <widget class="QLabel" name="MailLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="AdressLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adresse</string>
    </property>
   </widget>
   <widget class="QLabel" name="TelLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModifInfoButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier Information</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ModifMdpButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier Mot De Passe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GestionAssoButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestion De l'Asso</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>1191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableInfoEntreprise">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>291</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableDesProjEntreprise">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>831</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="validEntrepriseButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>240</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNewEnt">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>580</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouvelle Entreprise</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
